pipelines/kubeflow_pipelines_component_examples/run_notebook_component/README.md

```diff
@@ -4,7 +4,7 @@ Use the [_run notebook_ example component](https://github.com/elyra-ai/elyra/blo
 
 ### Prerequisites
 
-None. The example pipeline should work as is.
+To use this component, ensure that you create your pipeline with the accompanying `notebook` in your Jupyterlab workspace root directory.
 
```

Comment:
To review and run the example pipeline for this component, [clone the examples repository](https://github.com/elyra-ai/examples) and launch JupyterLab as shown below:

```
jupyter lab --NotebookApp.notebook_dir=examples/pipelines/kubeflow_pipelines_component_examples/run_notebook_component/
```